### ISSUE TITLE
Updating count in bottomNavigationMenu through setCount method is recreating fragment again

### ISSUE THREAD

**Issue by Dheeraj890** [SimformSolutionsPvtLtd/SSCustomBottomNavigation#50]

**Comment by Dheeraj890** [comment]
Could you help , how we can control  fragment to recreate when tvCount.text called on customNavigation class

**Comment by marioamg03** [comment]
Yeah, I had the same error. Then, when I updated the notification number in the BottomBar, my fragment was restarted.

viewModel.getNotificationCounter().observe(this) { count ->
    // Decidir el texto que se mostrará dependiendo del número de notificaciones no leídas
    val countText = when (count) {
        0 -> R.string.count_0
        1 -> R.string.count_1
        2 -> R.string.count_2
        3 -> R.string.count_3
        4 -> R.string.count_4
        5 -> R.string.count_5
        6 -> R.string.count_6
        7 -> R.string.count_7
        8 -> R.string.count_8
        9 -> R.string.count_9
        else -> R.string.count_9_plus // Para valores mayores a 9
    }

    // Asignar el texto a la vista del bottom navigation
    navView.setCount(3, countText)
}
